tests/wpt/web-platform-tests/html/editing/focus/document-level-focus-apis/document-level-apis.html

```diff
@@ -30,5 +30,9 @@
   assert_false(doc.hasFocus(), "The hasFocus() method should return false.");
 }, "The hasFocus() method must return false if the Document has no browsing context");
 
+test(function () {
+  assert_true(document.hasFocus(), "The hasFocus() method should return true.");
```

Comment:
I'm not sure whether it's always true that the browser has focus while WPT is running.  (Even if it works with the Servo test harness, it might fail for other browsers.)  This may be why this test was not included in this file originally.  It might be safest to revert this change.